cice.setup

```diff
@@ -786,6 +786,11 @@ EOF
       if ($bfbcomp != ${spval}) then
         set fbfbcomp = ${machcomp}_${bfbcomp_test_grid_pes}${bfbcomp_soptions}
       endif
+    else
```

Comment:
we are far below the ` if (${docase} == 0) then` line here, so for documentation purpose maybe a comment after the `else` would help to clarify where we are in the logic?

Reply:
oups, I l left this comment on an earlier commit of the series, the most recent does it differently so it's ok.

Reply:
This has actually changed in a more recent commit.  It's now outside the docase if test and establishes the standard.  It's only with suites that we invoke the more complex implementation.